db/migrate/20180521172343_create_offers.rb

```diff
@@ -0,0 +1,13 @@
+class CreateOffers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :offers do |t|
+      t.text :description
+      t.string :title
+      t.integer :suggested_price, :default => 0
+      t.string :status, :default => "Proposed"
```

Comment:
I would definitely use lowercase as a default value: `proposed` - someone can try to compare values or make a query to find records with a specific status value in the future - I think they would most probably look for all lowercase words as that is most common approach.

Reply:
Modified